discourse_migration/discourse-categories.md

```diff
@@ -0,0 +1,75 @@
+# Proposed Discourse Categories
+
+* Announcements
+
+* Releases
```

Comment:
Will this have anything other than announcements, how about putting Releases as a subcategory under Announcements?

If it isn't announce'y it would be good to put it somewhere else - we have a bunch of top-levels

Reply:
The Release  category would have the discussion among release testers which currently takes place on the release-testers list.  I also get a lot of questions about the releases asked to me either by direct email or on IRC/Discord, so my hope is that this category could be used for those.

Reply:
Ok, I'd recommend moving this under "Project Infrastructure" category then.

Reply:
I'm not really sure it should go under project infrastructure. It really has nothing to do with infrastructure. What is the negative with it being a top level category?

Reply:
Top level items take a lot of visible area on the [main page](https://llvm.discourse.group/).  Discussion between release testers seems extremely infra, or pontentially "community" level, not top level.

Reply:
Actually, here's what I'd recommend doing: I'd suggest renaming this to "Release Testing" and put it under Infrastructure.  If we want release announcements to be separate from general announcements then we can have a separate "Announcements / Releases" subcategory for them.